api/src/wfl/module/covid.clj

```diff
@@ -23,7 +24,51 @@
 (defn create-covid-workload!
   [tx {:keys [source sink executor] :as request}])
 
-;; TODO: implement progressive (private) functions inside this update loop
+(defn ^:private snapshot-created?
+  "Check if snapshot has been created in TDR, return snapshot id if so."
+  [{:keys [snapshot_job_id] :as _record}]
```

Comment:
Is the idea that this check would be called with a record from the `Sarscov2IlluminaFull` instance as the argument?